site/package.json

```diff
@@ -9,12 +9,12 @@
     "clean": "gulp clean"
   },
   "devDependencies": {
```

Comment:
@jdanielmyers I updated my packages and this apparently changed this file. Should this be the case?